packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1249,3 +1249,6 @@ export const IP_VERSION = {
 } as const;
 
 export type IP_VERSION = (typeof IP_VERSION)[keyof typeof IP_VERSION];
+
+// constant for if the permissionToken is about to expire in the next 10 seconds, refresh it
+export const MEETING_PERMISSION_TOKEN_REFRESH_THRESHOLD_IN_SEC = 10;
```

Comment:
I know UCF has it as 10s but maybe we should start with a bit bigger value, like 30s?